paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -659,24 +660,12 @@ class BroadcastDataMKLDNNHandler
           y->format(), MKLDNNMemoryFormat::undef,
           platform::errors::InvalidArgument("Wrong format set for Y tensor."));
 
-      auto src1_tz = framework::vectorize(y->dims());
       const auto src0_tz = framework::vectorize(x->dims());
```

Comment:
Maybe you could be so kind to unify way of naming of variables in this constructor please? Mixing x, y, src_0_tz, src1_md, input_dims is a bit misleading.

Reply:
You're right, I'll unify my naming conventions